header.h

```diff
@@ -0,0 +1,2 @@
+#include <stddef.h>
+#include "vendor/xatlas.h"
```

Comment:
Please move this file into a relevant folder rather then sticking it into the root.

Reply:
In case anyone else is here looking to integrate xatlas into their Rust project. I've created a fork which amounts to an almost total rewrite here: https://github.com/PaulWagener/xatlas-rs/
It features an API to the latest version of xatlas with more lifetime checks.